docs/essentials/interactions.md

```diff
@@ -82,3 +82,5 @@ The above example uses the `canvasElement` to scope your element queries to the
 While you can refer to the [Testing Library documentation](https://testing-library.com/docs/) for details on how to use it, there's an important detail that's different when using the Storybook wrapper: **method invocations must be `await`-ed**. It allows you to step back and forth through your interactions using the debugger.
 
 Any `args` that have been marked as an Action, either using the [argTypes annotation](./actions.md#action-argtype-annotation) or the [argTypesRegex](./actions.md#automatically-matching-args), will be automatically converted to a [Jest mock function](https://jestjs.io/docs/mock-function-api) (spy). This allows you to make assertions about calls to these functions.
+
+> Note: If you need to use jest native spys like `jest.fn()` the import of jest over `import {jest} from "@storybook/jest"` is needed, as jest is not available globally.
```

Comment:
@pascalfiv if you're ok with it, could you instead adjust this to the following:

```md
<div class="aside">

ℹ️ To mock functions in your Storybook stories for reliable and isolated component testing, use the `jest` import from `@storybook/jest`. This allows you to avoid configuring Jest globally in your project.

</div>
```

Reply:
sure i changed it